### DISCUSSION TITLE
Dataview functionality issues

### DISCUSSION THREAD

**Discussion by andrewbartnik** [marcusolsson/obsidian-projects#335] [Help] [answered]
When using Dataview to query for notes that I want to include in a project, I have noticed some issues that aren't a problem when using folders. The completed fields aren't toggle-able(?) in the table view, and they don't check off in the calendar view. My query looks like this:

TABLE date as "Date", class as "Class", status as "Status", completed as "Completed"
FROM ("Projects")
WHERE order = "Assignment"

Is there something Im missing here?

**Comment by marcusolsson** [comment, marked as answer]
Hi and thanks for trying out Projects!

Unfortunately, due to technical limitations, Dataview projects are read-only for now.

**Reply by andrewbartnik** [reply]
Of course, its an amazing plugin - my new favorite!

When I use the folders to specify my projects directory, each time I add a new note that doesn't have a date frontmatter field, the date field in my each of my project views gets turned into a text field. My currrent workaround is to go manually exclude each note that doesn't have a date field. Is there a better way to do this?

**Reply by marcusolsson** [reply]
Could you create an issue for this, with some basic steps to reproduce?
